contribute/style-guides/themes.md

```diff
@@ -43,6 +43,36 @@ const Foo: React.FunctionComponent<FooProps> = () => ...
 export default withTheme(Foo);
 ```
 
+### Testing components that use ThemeContext
+
+When implementing snapshot tests of components that use theme (i.e. component that uses `withTheme` HOC), the snapshot will contain the entire theme object. This makes working with such snapshot hard as any change to the theme will result in the snapshot being outdated.
```

Comment:
```suggestion
When implementing snapshot tests for themable components, i.e. any component that uses the `withTheme` HOC, the snapshot will contain the entire theme object. This complicates working with such snapshots, as any change to the theme will result in the snapshot being outdated.
```

Not sure about this one though. What are your thought @oddlittlebird?

Reply:
@marcusolsson - I simplified the language, hopefully that helps. The word "outdated" is problematic. What is the actual result of the snapshot not matching the theme because the theme was changed? Also, what is the message here? Are we trying to warn the user not to do that? If so, the language needs to be stronger.

Reply:
@oddlittlebird it's an informative paragraph introducing some motivation for mocking the ThemeContext